src/mailmojo-widget.php

```diff
@@ -32,13 +37,14 @@ public function __construct() {
 		parent::__construct('mailmojo', __('MailMojo Signup Form', 'mailmojo'), $options);
 
 		add_action('init', array($this, 'initFiles'));
-		add_action('parse_request', array($this, 'subscribe'));
 
 		$this->plugin = MailMojoPlugin::getInstance();
 	}
 
 	/**
-	 * Inits the javascript, css and localization files.
+	 * Initiates the CSS file.
```

Comment:
Suggest "Adds our custom CSS file.", the word "initiate" doesn't really work here.